service/controller/app/v1/resource/kubeconfigfinalizer/resource.go

```diff
@@ -0,0 +1,44 @@
+package kubeconfigfinalizer
+
+import (
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	Name = "kubeconfigv1"
```

Comment:
Just noticed that the resource name is wrong. It should be `kubeconfigfinalizerv1`.

Reply:
@tomahawk28 Can you update this pls?

Reply:
https://github.com/giantswarm/app-operator/pull/98 Sure, I just added

Reply:
@tomahawk28 Thx! ❤️